contracts/DaiHrd.sol

```diff
@@ -27,7 +28,7 @@ import { RuntimeConstants } from "./RuntimeConstants.sol";
  * are no special restrictions in the amount of tokens that created, moved, or
  * destroyed. This makes integration with ERC20 applications seamless.
  */
-contract ERC777 is RuntimeConstants, Context, IERC777, IERC20 {
+contract ERC777 is Context, IERC777, IERC20, RuntimeConstants {
```

Comment:
Out of curiosity, why the change in order here?

Reply:
accident, i played around with changing hierarchy, put it back in the wrong order, i'll fix (although no-op, since there's no storage defined there)